configs/genesis-assets/genesis.json

```diff
@@ -118,6 +118,10 @@
       "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
       "0x0372ee6669dc86563007245ed7343d5180b96221ce28f44408cff2898038dbd4"
     ],
+    [
+      "0x4718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d",
+      "0x0372ee6669dc86563007245ed7343d5180b96221ce28f44408cff2898038dbd4"
```

Comment:
Can you confirm, "0x4718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d" is the address of a ERC20 contract "0x0372ee6669dc86563007245ed7343d5180b96221ce28f44408cff2898038dbd4", pre-deployed?

Reply:
What do you mean by "pre-deployed" ? In Madara I understand the genesis.json content is deployed at start.
I took the Madara ETH address, i see it's the same on the Starknet mainnet, so i take the STRK address on the mainnet and i set it on genesis.
I misunderstand something ?

Reply:
It's fine, I think it can work